src/Service.Tests/Authentication/JwtTokenAuthenticationUnitTests.cs

```diff
@@ -215,22 +215,36 @@ public async Task TestInvalidToken_InvalidSigningKey()
         }
 
         /// <summary>
-        /// JWT with intentionally scrambled signature should result in failed authentication
-        /// characterized with an HTTP 401 Unauthorized response.
-        /// JWT signed with RSASecurityKey, not a cert, so no KID (keyID) claim in token.
+        /// JWT signed with key not registered in the server should result in failed authentication
+        /// characterized with an HTTP 401 Unauthorized response due to an "invalid signature" error.
+        /// If this test fails, check the console output for the error:
+        /// "Bearer was not authenticated. Failure message: IDX10503: Signature validation failed."
         /// </summary>
         [TestMethod("JWT signed with unrecognized/unconfigured key, results in signature key not found")]
         public async Task TestInvalidToken_InvalidSignature()
         {
-            RsaSecurityKey key = new(RSA.Create(2048));
-            string token = CreateJwt(audience: AUDIENCE, issuer: LOCAL_ISSUER, signingKey: key);
-            string tokenForgedSignature = ModifySignature(token, removeSig: false);
-
-            HttpContext postMiddlewareContext = await SendRequestAndGetHttpContextState(key, tokenForgedSignature);
+            // Arrange
+            RsaSecurityKey tokenIssuerSigningKey = CreateRsaSigningKeyForTest();
```

Comment:
Which function necessitated this test change for .NET 8 upgrade?

Reply:
Upgrading to dotnet8 required that I update `System.IdentityModel.Tokens.Jwt`, which transitively depends on newer version of `Microsoft.IdentityModel.Tokens.Jwt` that had a breaking change requiring stricter parameter values on the RsaKey object created for signing jwt tokens in the test fixture.